client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -178,6 +178,7 @@ export interface SiteData {
 	isFavorite?: boolean;
 	isSelected?: boolean;
 	onSelect?: () => void;
+	ref: string | number;
```

Comment:
```suggestion
	ref?: string | number;
```
I forgot to mark this field as "optional" to avoid TypeScript complains